_posts/2018-08-13-Introducing....md

```diff
@@ -0,0 +1,96 @@
+
+As a data scientist, her job is analytical, and involves things like cleaning data sets, chosing the right methods to answer particular research questions, and creating data visualisations. As a software engineer, her work involves building tools, and following best practices when using code to answer questions. She says that it’s very important to remember that code is not infallible, a human has to create it and we are subject to our own biases and opinions that might be transferred.
+
+If you don’t follow best practice, you run the risk of creating unreproducible code, or code that doesn’t do what you think it’s doing. There is the risk of the code producing an answer or output that that isn’t obviously wrong but is still wrong, and this is a real issue if there’s no transparency and no-one can interrogate it.
+
+Sarah is involved in creating The Turing Way. This was originally a guide for doing reproducible data science; but it is evolving into a more general guide. She has been linking up with the Binder community, as they have a great tool for sharing code and data very easily. Binder is a solution to being able to test whether you can get the same results from the same data: this could be by using mybinder.org which is completely public, or by installing your own binder hub and enabling it to work with a private repository. 
+
+#### “What’s your ideal outcome of your time at the Turing?”
+
+Sarah finds that the Turing gives her the opportunity to work in a lot of different domains and projects. Her ideal outcome would be that through the Turing she can “find my groove” as well as the domain that she is best at and appeals to her most. She’s proud to work with the Binder community and has been asked to be a long-term contributor - anything she could do to contribute to the community and push technology forward would make her very proud. 
+
+Sarah also wants to learn to do all of the cool things - every day is a different challenge, and it’s nice to have space to learn things properly!
+
+#### “How are you finding the Turing?”
+
+“Amazing! It’s a really supportive environment, and the ethos is much more about collaboration than competition. People are not too possessive over their work. It’s nice to be able to share knowledge and get to a good solution together.”
+
+## :star: Ang Li :star: 
+
+#### “Tell me about your background”
+
+Ang is currently a PhD student. His college major was in engineering, specialising in space flight. He studied in the Northeast of China in the Heliongjiang Province at the University of Harbin. Ang was born in a very small city. At first, he didn’t know that much about engineering, but he was told that it was very lucrative! When he started studying, however, he found that he was not so interested in engineering and switched to brain science. He is very interested in the biology of the brain.
+
```

Comment:
He studied in the Northeast of China in the **_Heilongjiang_** Province **at the Harbin Institute of Technology**. Ang was born in a very small city. At first, he didn’t know that much about engineering, but he was told that it was very lucrative! When he started studying, however, he found that he was not so interested in engineering and switched to brain science. He is very interested in the biology of the brain.